### PR TITLE
add default path as locate not reliable

### PR DESCRIPTION
**Pull request by remicollet**
On a fresh install, locate returns nothing, thus making nearly impossible to find the library, espcially in minimal build system.

### CODE REVIEW

**Comment by rtheunissen** [comment]
Thanks! I was making it up as I went so I appreciate the help. I had to refresh the locate db on Travis to get tests passing...

File: config.m4 [hunk 1]
```diff
@@ -4,7 +4,7 @@ PHP_ARG_ENABLE(decimal, whether to enable decimal support,
 if test "$PHP_DECIMAL" != "no"; then
 
     AC_MSG_CHECKING([for libmpdec files in default path])
-    for i in $(locate -e /usr/**/libmpdec.so); do
+    for i in /usr/$PHP_LIBDIR $(locate -e /usr/**/libmpdec.so); do
```
Comment (remicollet) [inline config.m4]: In fact this is wrong.... /usr/lib always exists, so this doesn't detect anything ;)

Will submit another PR
